server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigParser.java

```diff
@@ -412,7 +379,13 @@ private void validateGatewayConfig(GatewayConfigDocument configDoc, List<String>
                         LOGGER.error("  Line: " + line + " Column: " + column);
                     }
                 }
-                LOGGER.error("  " + error.getMessage().replaceAll("@" + GATEWAY_CONFIG_NS, ""));
+                LOGGER.error("  " + error.getMessage().replaceAll("@" + GatewayConfigNamespace.CURRENT_NS, ""));
+                if (error.getMessage().contains("notify-options")) {
```

Comment:
Yeah, suggesting if (error.getMessage().contains("notify-options") || error.getMessage().contains("notify")).  Because both those elements are specific to APNS, and it's not clear to me which exception would get thrown first